install.sh

```diff
@@ -321,177 +368,177 @@ if ($INSTALL_SUB) && !(ls $CATKIN_DIR/src | grep --quiet "Sub8"); then
-	apt-get source --compile -qq python-pyode
-	sudo dpkg -i python-pyode_*.deb
+	# Software to interface with MIL hardware
+	ros_git_get https://github.com/uf-mil-archive/hardware-common
 
-	# Message types
-	sudo apt-get install -qq ros-indigo-tf2-sensor-msgs ros-indigo-tf2-geometry-msgs
+	# Required steps to build and install lqRRT
+	ros_git_get https://github.com/jnez71/lqRRT.git
+	sudo python $CATKIN_DIR/src/lqRRT/setup.py build
+	sudo python $CATKIN_DIR/src/lqRRT/setup.py install
 
-	# Pulling large project files from Git-LFS
+	# Pull large project files from Git-LFS
 	instlog "Pulling large files for Navigator"
 	cd $CATKIN_DIR/src/Navigator
 	git lfs pull
-
-	instlog "Cloning Navigator Git repositories that need to be built"
-	ros_git_get https://github.com/jnez71/lqRRT.git
-	ros_git_get https://github.com/gareth-cross/rviz_satellite.git
-
-	# Required steps to build and install lqRRT
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py build
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py install
 fi
 
 
-#==============================#
-# Sub8 Dependency Installation #
-#==============================#
-
-if ($INSTALL_SUB); then
-	instlog "Installing Sub8 dependencies from the Ubuntu repositories"
-
-	# Optical character recognition
-	sudo apt-get install -qq tesseract-ocr
-
-	# Hardware drivers
-	sudo apt-get install -qq libusb-1.0-0-dev
-
-	instlog "Installing Sub8 ROS dependencies"
-
-	# 3D Mouse
-	sudo apt-get install -qq ros-indigo-spacenav-node
-
-	instlog "Installing Sub8 dependencies from Python PIP"
-
-	# Libraries needed by the hydrophone board
-	sudo pip install -q -U crc16
-fi
-
 #=========================#
 # Bashrc Alias Management #
 #=========================#
 
-# Copies the MIL aliases file to the machine's home directory file and sources it
-cat $CATKIN_DIR/src/Navigator/scripts/mil_aliases.sh > $ALIASES_FILE
-source $ALIASES_FILE
-if !(cat $BASHRC_FILE | grep --quiet "source $ALIASES_FILE"); then
-	echo "source $ALIASES_FILE" >> $BASHRC_FILE
+SWC_ALIASES=$CATKIN_DIR/src/software-common/scripts/bash_aliases.sh
+SUB_ALIASES=$CATKIN_DIR/src/Sub8/scripts/bash_aliases.sh
+NAV_ALIASES=$CATKIN_DIR/src/Navigator/scripts/bash_aliases.sh
+
+# Source the software-common configurations for bash on this user account
+source $SWC_ALIASES
+if !(cat $BASHRC_FILE | grep --quiet "source $SWC_ALIASES"); then
+	echo "" >> $BASHRC_FILE
+	echo "# Adds MIL aliases to shell environment" >> $BASHRC_FILE
+	echo "source $SWC_ALIASES" >> $BASHRC_FILE
+fi
+
+# Source the Sub8 configurations for bash on this user account
+if ($INSTALL_SUB); then
+	source $SUB_ALIASES
+	if !(cat $BASHRC_FILE | grep --quiet "source $SUB_ALIASES"); then
+		echo "" >> $BASHRC_FILE
+		echo "# Adds SubjuGator aliases to shell environment" >> $BASHRC_FILE
+		echo "source $SUB_ALIASES" >> $BASHRC_FILE
+	fi
 fi
 
-instlog "The ~/.mil_aliases file has been updated with the latest aliases"
+# Source the Navigator configurations for bash on this user account
```

Comment:
**_Idea:_** we create a .milrc in the user's home directory so all the mil repos only have to work with that file and there's only one line that has to be sourced from the bashrc.  It's  much easier to modify a file of our own design than keep appending to the user's.